dev-php/phpunit/files/autoload-8.5.0.php

```diff
@@ -225,8 +225,8 @@
 'phpunit\\util\\xml' => '/Util/Xml.php',
 'phpunit\\util\\xmltestlistrenderer' => '/Util/XmlTestListRenderer.php',
 ],
-  "${include_dir}/PHPUnit"
-);
+  __DIR__ . '/../src/'
+ );
```

Comment:
Here, you're modifying a file which existed for previous revisions of the package.

Usually it's a good idea to copy the old filename to e.g. autoload-8.5.0-r1.php or whatever your new version is - and use the new filename in your ebuilds, so that old versions continue to work, in case there's a problem with your change.

Reply:
Yes that is true, I did not wanted to change existing files,
I am putting my changes on 8.5.2-autoload{,-test}.php